library/locale.po

```diff
@@ -60,18 +70,32 @@ msgid ""
 "exception :exc:`Error` is raised. If successful, the new locale setting is "
 "returned."
 msgstr ""
+"Si *locale* ne vaut pas ``None``, :func:`setlocale` modifie le paramètre "
+"locale pour la catégorie *category*. Les catégories disponibles sont listés "
+"dans la description des données ci-dessous. *locale* peut être une chaîne de "
+"caractères ou un itérable de deux chaînes de caractères (code de la langue "
+"et encodage). Si c'est un itérable, il est converti en un nom de locale à "
+"l'aide du moteur d'aliasing de locale. Si c'est une chaîne vide, les "
+"paramètres par défaut de l'utilisateur sont utilisés. Si la modification de "
+"la locale échoue, l'exception :exc:`Error` est levée. Si elle fonctionne, la "
+"nouvelle locale est renvoyée."
 
 #: ../Doc/library/locale.rst:43
 msgid ""
 "If *locale* is omitted or ``None``, the current setting for *category* is "
 "returned."
 msgstr ""
+"Si *locale* est omis ou vaut ``None``, le paramètre actuel de *category* est "
+"renvoyé."
 
 #: ../Doc/library/locale.rst:46
 msgid ""
 ":func:`setlocale` is not thread-safe on most systems. Applications typically "
 "start with a call of ::"
 msgstr ""
+":func:`setlocale` n'est pas sécurisé pour les appels concurrents (*thread-"
```

Comment:
I'm not found of "sécurisé pour les appels concurrents", yet I have no other idea, maybe keep thread-safe? https://fr.wikipedia.org/wiki/Thread_safety

Reply:
I'll keep *thread-safe*, even if what christopheNan suggested seems attractive.